Gemfile.lock

```diff
@@ -0,0 +1,211 @@
+      opentelemetry-common (~> 0.19.3)
+      opentelemetry-registry (~> 0.2)
+      opentelemetry-semantic_conventions
+    opentelemetry-semantic_conventions (1.8.0)
+      opentelemetry-api (~> 1.0)
+    os (1.1.4)
+    parallel (1.22.1)
+    parser (3.2.0.0)
+      ast (~> 2.4.1)
+    pickup (0.0.11)
+    prometheus-client (2.1.0)
+    public_suffix (5.0.1)
+    puma (5.6.5-java)
+      nio4r (~> 2.0)
+    rack (2.2.6.4)
+    rack-protection (2.2.4)
+      rack
+    rainbow (3.1.1)
+    rake (13.0.6)
+    redis (4.8.0)
+    regexp_parser (2.6.2)
+    representable (3.2.0)
+      declarative (< 0.1.0)
+      trailblazer-option (>= 0.1.1, < 0.2.0)
+      uber (< 0.2.0)
+    retriable (3.1.2)
+    rexml (3.2.5)
+    rspec (3.12.0)
+      rspec-core (~> 3.12.0)
+      rspec-expectations (~> 3.12.0)
+      rspec-mocks (~> 3.12.0)
+    rspec-core (3.12.0)
+      rspec-support (~> 3.12.0)
+    rspec-expectations (3.12.2)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.12.0)
+    rspec-mocks (3.12.4)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.12.0)
+    rspec-support (3.12.0)
+    rubocop (1.1.0)
+      parallel (~> 1.10)
+      parser (>= 2.7.1.5)
+      rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.8)
+      rexml
+      rubocop-ast (>= 1.0.1)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 2.0)
+    rubocop-ast (1.24.1)
+      parser (>= 3.1.1.0)
+    ruby-progressbar (1.11.0)
+    ruby2_keywords (0.0.5)
+    signet (0.17.0)
+      addressable (~> 2.8)
+      faraday (>= 0.17.5, < 3.a)
+      jwt (>= 1.5, < 3.0)
+      multi_json (~> 1.10)
+    simplecov (0.22.0)
+      docile (~> 1.1)
+      simplecov-html (~> 0.11)
+      simplecov_json_formatter (~> 0.1)
+    simplecov-html (0.12.3)
+    simplecov_json_formatter (0.1.4)
+    sinatra (2.2.4)
+      mustermann (~> 2.0)
+      rack (~> 2.2)
+      rack-protection (= 2.2.4)
+      tilt (~> 2.0)
+    spicy-proton (2.1.15)
+      bindata (~> 2.3)
+    statsd-ruby (1.5.0)
+    thrift (0.17.0)
+    tilt (2.0.11)
+    trailblazer-option (0.1.2)
+    uber (0.1.0)
+    unicode-display_width (1.8.0)
+    vmpooler (3.0.0)
+      concurrent-ruby (~> 1.1)
+      connection_pool (~> 2.2)
+      deep_merge (~> 1.2)
+      net-ldap (~> 0.16)
+      opentelemetry-exporter-jaeger (= 0.20.1)
+      opentelemetry-instrumentation-concurrent_ruby (= 0.19.2)
+      opentelemetry-instrumentation-http_client (= 0.19.4)
+      opentelemetry-instrumentation-redis (= 0.21.3)
+      opentelemetry-instrumentation-sinatra (= 0.19.3)
+      opentelemetry-resource_detectors (= 0.19.1)
+      opentelemetry-sdk (~> 1.0, >= 1.0.2)
+      pickup (~> 0.0.11)
+      prometheus-client (~> 2.0)
+      puma (~> 5.0, >= 5.0.4)
+      rack (~> 2.2)
+      rake (~> 13.0)
+      redis (~> 4.1)
+      sinatra (~> 2.0)
+      spicy-proton (~> 2.1)
+      statsd-ruby (~> 1.4)
+    webrick (1.8.1)
+    zonefile (1.06)
+
+PLATFORMS
+  universal-java-1.8
+  universal-java-11
```

Comment:
In https://github.com/puppetlabs/vmpooler-provider-vsphere/pull/29/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL175 you remove one of the platforms... do these need to be consistent across repos?

Reply:
That must be leftover from when I started with jruby 9.3 then updated to 9.4 and didn't remove the old, so I'll remove that.